experimental/QuadFormAndIsom/test/runtests.jl

```diff
@@ -0,0 +1,261 @@
+using Test
+using Oscar
+
+@testset "Printings" begin
+  function _show_details(io::IO, X::Union{ZZLatWithIsom, QuadSpaceWithIsom})
+    return show(io, MIME"text/plain"(), X)
+  end
+  L = root_lattice(:A, 2)
+  Lf = integer_lattice_with_isometry(L)
+  Vf = ambient_space(Lf)
+  for X in [Lf, Vf]
+    @test sprint(_show_details, X) isa String
+    @test sprint(Oscar.to_oscar, X) isa String
+    @test sprint(show, X) isa String
+    @test sprint(show, X, context=:supercompact => true) isa String
```

Comment:
this doesn't really test anything. doctests seem to be a better method to test printings

Reply:
It does test that there are no mistake. I added those for the test coverage since doctests do not play a part there. I would rather keep it as a double check that there no problems with the printing.

Reply:
Doctests (on julia-1.9) are indeed taken into account for coverage. See https://github.com/oscar-system/Oscar.jl/blob/d0f7ea722b4a83fb525b1e9a2941958f6ee0bb31/.github/workflows/CI.yml#L127

Furthermore, the only thing you test here is the absence of an error. If you want to test something sensible, you could compare the `sprint` against a fixed string.

Reply:
I think the smoke test is fine (testing that printing that does not error). I would not compare to some fixed string.